test/Python/src/ScheduleValue.lf

```diff
@@ -11,7 +11,7 @@ main reactor ScheduleValue {
     reaction(a) {=
         print("Received: ", a.value)
         if a.value != "Hello":
-            sys.stderr.write('FAILURE: Should have received "Hello"\n')
+            sys.stderr.write("FAILURE: Should have received 'Hello'\n")
```

Comment:
Here `"'...'" seems to work. Is this correct?

Reply:
Yes this works, because the double-quoted string production also consumes `'` fine